core-libs/setup/ssr/error-handling/express-error-handlers/express-error-handlers.ts

```diff
@@ -0,0 +1,33 @@
+/*
+ * SPDX-FileCopyrightText: 2024 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import {
+  CmsPageNotFoundOutboundHttpError,
+  HttpResponseStatus,
+} from '@spartacus/core';
+import { ErrorRequestHandler } from 'express';
+
+/**
+ * Returns default handlers which results in a fallback to client side rendering.
+ * - If cms page not found, the document content is sent to the client with the appropriate 404 status code.
+ * - For rest of errors, the document content is sent to the client with the appropriate status 500 code.
+ *
+ * @param documentContent The document content to be sent to the client.
+ * @returns The error request handler.
+ */
+//EXPRESS JS ERROR HANDLER
+export const defaultExpressErrorHandlers =
```

Comment:
[MINOR]
For better dicoverability in IDE, let's align the file name to `default-express-error-handlers.ts`

Reply:
I thought about it, but decided to leave the file name more generic to avoid being closed to possible addition of new handlers.

Reply:
We're never closed with filenames, you can always rename it.
Now we have one function, we can create one file with its name for better discoverability.
I wouldn't mind if we try to use the rule "one thing - one file"  as it helps finding things quicker in IDE via quick search by name (e.g `cmd+P`)